scenariolib/visit.go

```diff
@@ -134,11 +134,16 @@ func (v *Visit) ExecuteScenario(scenario Scenario, c *Config) error {
 		if v.WaitBetweenActions {
 			var timeToWait int
 			if c.TimeBetweenActions > 0 {
-				timeToWait = c.TimeBetweenActions
+				timeToWait = c.TimeBetweenActions //Random waiting time
+				WaitBetweenActions(timeToWait)
+			}else if c.TimeBetweenActions < 0 {
+				timeToWait = c.TimeBetweenActions*(-1)
+				time.Sleep(time.Duration(timeToWait) * time.Second) //Constant Waiting time
```

Comment:
Using a small number ends up not waiting at all too often, since the random generator can return 0. But, the main reason is that it is simpler to control the total QPS of all the bots if we know exactly how much time each bot waits between actions.